File: Sistema/src/app/models/ConsultaUsuarioLoja.Model.ts
export class ConsultaUsuarioLojaModel {
  objUsuarioLoja: UsrLoja;
  objLojaConsulta: LojaConsulta;

  constructor() {
    this.objUsuarioLoja = new UsrLoja();
    this.objLojaConsulta = new LojaConsulta();
  }
}

export class UsrLoja {
  UsuCodi: number;
  UsuNome: string;
  UsuNCIM: string;
  UsuNasc: Date;
  UsuEmai: string;
  UsuNCel: string;
  UsuStat: boolean;
  LojNome: string;
  LojNumL: string;
  PotCodi: number;

  constructor() {
    this.UsuCodi = 0;
    this.UsuNome = '';
    this.UsuNCIM = '';
    this.UsuNasc = new Date();
    this.UsuEmai = '';
    this.UsuNCel = '';
    this.UsuStat = false;
    this.LojNome = '';
    this.LojNumL = '';
    this.PotCodi = 0;
  }
}

export class LojaConsulta {
  LojCodi: number;
  LojNome: string;
  LojNumL: string;
  LojLogo: string;
  LojLogr: string;
  LojNume: string;
  LojBair: string;
  LojStat: boolean;
  CidCodi: number;
  PotCodi: number;
  RitCodi: number;

  constructor() {
    this.LojCodi = 0;
    this.LojNome = '';
    this.LojNumL = '';
    this.LojLogo = '';
    this.LojLogr = '';
    this.LojNume = '';
    this.LojBair = '';
    this.LojStat = false;
    this.CidCodi = 0;
    this.PotCodi = 0;
    this.RitCodi = 0;
  }
}

File: API/Models/ConsultaUsuarioLojaModel.cs
namespace API_Visitatus.Models
{
    public class ConsultaUsuarioLojaModel
    {
        public long sesCodi { get; set; }
        public UsrLoja? objUsuarioLoja { get; set; }
        public LojaConsulta? objLojaConsulta { get; set; }

        public class UsrLoja
        {
            public long UsuCodi { get; set; }
            public string? UsuNome { get; set; }
            public string? UsuNCIM { get; set; }
            public DateTime UsuNasc { get; set; }
            public string? UsuEmai { get; set; }
            public string? UsuNCel { get; set; }
            public bool UsuStat { get; set; }
            public string? LojNome { get; set; }
            public string? LojNumL { get; set; }
            public int PotCodi { get; set; }
        }

        public class LojaConsulta
        {
            public long LojCodi { get; set; }
            public string? LojNome { get; set; }
            public string? LojNumL { get; set; }
            public string? LojLogo { get; set; }
            public string? LojLogr { get; set; }
            public string? LojNume { get; set; }
            public string? LojBair { get; set; }
            public bool LojStat { get; set; }
            public long? CidCodi { get; set; }
            public int PotCodi { get; set; }
            public int? RitCodi { get; set; }
        }
    }
}
